src/burp/backup.epp

```diff
@@ -1617,68 +1617,152 @@ void put_index( burp_rel* relation)
 	// requests--this requires more code but it is well worth it
 	// for the performance benefits, especially remotely--deej
 
-	FOR (REQUEST_HANDLE tdgbl->handles_put_index_req_handle1)
-		X IN RDB$INDICES WITH
-		X.RDB$RELATION_NAME EQ relation->rel_name
+	if (tdgbl->runtimeODS >= DB_VERSION_DDL13_1)
```

Comment:
Diff comparation is not easy here.

Can't we use unique code where non-existent fields are read as NULL?

Or are you preventing problem of new gbak being connected to very old engine that treat non-existent fields as error?

Reply:
Yes, this is to please old engine versions.